src/ja/common/message/worker_commands/get_workload.py

```diff
@@ -0,0 +1,31 @@
+"""
+This command will get the cpu_load and other information about the state of the work machine
+"""
+from typing import Dict
+
+from ja.common.message.base import Serializable
+from ja.common.message.worker import WorkerCommand, WorkerResponse
+from ja.worker.worker_client import Worker
+
+
+class CancelJobCommand(WorkerCommand):
```

Comment:
You forgot to change the name of the class to QueryCommand, or whatever you want to call it.

Reply:
I'm not even good at copy pasting.